packages/eslint-plugin/src/rules/local-cx-xcss/README.md

```diff
@@ -0,0 +1,31 @@
+# `local-cx-xcss`
+
+This rule ensures the `cx()` function is only used within the `xcss` prop. This aids tracking what styles are applied to a jsx element.
+
+👎 Examples of **incorrect** code for this rule:
+
+```js
+import { cx, styleMap } from '@compiled/react';
```

Comment:
`styleMap` is `cssMap` as per https://github.com/atlassian-labs/compiled/blob/master/packages/react/src/index.ts#L12

Reply:
Thanks will fix up